tools/generate_configuration.cpp

```diff
@@ -105,8 +105,25 @@ void GenerateConfiguration(std::string const& game_epoch,
                                                                    : name)
                       << "\n";
     if (body.has_gravitational_parameter()) {
-      gravity_model_cfg << "    gravitational_parameter = "
-                        << body.gravitational_parameter() << "\n";
+      // TODO(phl): We don't properly marshall non-ASCII strings in structs, so
+      // if the gravity model has references to the solar or terrestrial masses,
+      // switch to SI units.
+      if (body.gravitational_parameter().find_first_of(u8"☉🜨") ==
+          std::string::npos) {
+        gravity_model_cfg << "    gravitational_parameter = "
+                          << body.gravitational_parameter() << "\n";
+      } else {
+        GravitationalParameter const gravitational_parameter =
+            ParseQuantity<GravitationalParameter>(
+                body.gravitational_parameter());
+        gravity_model_cfg << "    gravitational_parameter = "
+                          << std::scientific
```

Comment:
`DebugString(gravitational_parameter)` (in SI units, rather than multiples of SI units) would avoid losing an ULP from the division by 1 000 000 000.

Reply:
We use `km^3/s^2` everywhere, now is not the time to change this.